terraform/workspace-variables/app_config.yml

```diff
@@ -35,7 +35,7 @@ development:
 # ey-recovery-content
 content:
   <<: *defaults
-  # TRAINING_MODULES: demo-modules
+  TRAINING_MODULES: demo-modules
```

Comment:
Is this intended? I can't remember what this change will be in practice

Reply:
It means review apps use demo content not genuine

Reply:
Yeah I thought so. Will the content team want that? Don't they use review apps to see their content and will it confuse things?

Reply:
content team needs real content, so yea, should keep this commented out

Reply:
I'll change it but I think we should default to demo content and activate genuine as required to ensure ours stays up to date. This was the intent when I introduced it, the content shouldn't matter.

Reply:
I'm happy with that too, but only once we have a simple mechanism to activate genuine